.ci/ansible/tasks/setup_test_script_windows.yml

```diff
@@ -124,6 +129,8 @@
 
        Add-Content C:\Users\ogc\.op\.env  "${REPORT}"
 
-       $env:TAGS="{{ lookup('env', 'TAGS') }}"; $env:FORMAT="pretty,cucumber:${REPORT}.json,junit:${REPORT}.xml"; make --no-print-directory -C "{{ e2e_base_dir }}e2e/_suites/${SUITE}" functional-test-win
+       $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine")
+       cd "{{ e2e_base_dir }}\e2e\_suites\${SUITE}"
+       go test -timeout 60m -v --godog.format="pretty,cucumber:${REPORT}.json,junit:${REPORT}.xml" --godog.tags="{{ lookup('env', 'TAGS') }}"
```

Comment:
IIRC for some reason I was not able to propagate the environment so that running the Makefile caused the execution to fail because `go` was not found. With this, because the `env:Path` is set, it fairly works.